mautrix_facebook/portal.py

```diff
@@ -573,7 +574,22 @@ async def handle_matrix_message(self, sender: 'u.User', message: MessageEventCon
                                     event_id: EventID) -> None:
         # TODO handle errors?
         # try:
```

Comment:
This TODO and the corresponding commented `except` block below are done now

Reply:
It seems like they are dealing with specific error types? Is there any benefit to keeping that around at all?

Reply:
No, it's legacy fbchat stuff (pre mobile api rewrite)